packages/components/tests/dummy/app/templates/components/tooltip.hbs

```diff
@@ -0,0 +1,155 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Tooltip Component"}}
+
+<Shw::Text::H1>Tooltip</Shw::Text::H1>
+
+<section data-test-percy>
```

Comment:
I'll just leave the attribute as is since it was already in the template I believe and it's used in other showcases.

I'll work on adding mock classes and let you know if I have trouble, thanks.

Reply:
The only real states with any visible effect for this component are "focus" and "hover". I added a mock focus style for the button in the usual way.

For the "hover" state, the display of the tooltip is controlled by Tippy.js JavaScript code. The only way I could think to replicate it was to copy and paste the tooltip into the DOM for the "hover" example. I don't actually think we want to do this but I included it for now just so you can see...

Reply:
Indeed, more difficult to mock hover. I'm fine with just having focus, it's helpful for visual regression.

Reply:
Thanks. 

@didoo what do you think?

Reply:
Thanks, I got this to work.